js/gate.js

```diff
@@ -2823,7 +2845,7 @@ module.exports = class gate extends Exchange {
             request['chain'] = network;
             params = this.omit (params, 'network');
         }
-        const response = await this.privateWithdrawalsPost (this.extend (request, params));
+        const response = await this.privateWithdrawalsWithdrawalsPost (this.extend (request, params));
```

Comment:
@sc0Vu I think you have a typo here, should be `privateWithdrawalsPostWithdrawals` right?

Reply:
You're right, fixed!